packages/core/src/auth/IGetWorkflowsOptions.ts

```diff
@@ -0,0 +1,63 @@
+/*
```

Comment:
This file should be in `packages/workflows/src/doc`, not in the core SDK.

Reply:
fixed in commit